backend/inspirehep/records/marshmallow/literature/es.py

```diff
@@ -166,6 +170,22 @@ def get_bookautocomplete(self, record):
 
         return {"input": input_values}
 
+    def get_oai(self, record):
+        if is_cds_set(record):
+
+            return {
+                "id": f"oai:inspirehep.net:{record['control_number']}",
+                "sets": [current_app.config["OAI_SET_CDS"]],
+                "updated": record.updated,
+            }
+        elif is_cern_arxiv_set(record):
+            return {
+                "id": f"oai:inspirehep.net:{record['control_number']}",
+                "sets": [current_app.config["OAI_SET_CERN_ARXIV"]],
+                "updated": record.updated,
+            }
```

Comment:
what if it's in both? I don't think they're mutually exclusive: https://old.inspirehep.net/search?ln=en&ln=en&p=909COp%3ACERN%3Aarxiv+and+909COp%3AForCDS&of=hb&action_search=Search&sf=earliestdate&so=d&rm=&rg=25&sc=0.